artemis-journal/src/main/java/org/apache/activemq/artemis/core/journal/impl/dataformat/JournalAddRecord.java

```diff
@@ -30,32 +30,37 @@
 
    protected final byte recordType;
 
-   protected final boolean add;
+   protected final byte journalType;
```

Comment:
Is this back compatible?

Reply:
It is... Compatibility tests would be failing if it wasn't.


addBytes will write 1 for true, 0 for false.

Reply:
Cool just checking